bot/exts/utilities/githubinfo.py

```diff
@@ -78,6 +81,21 @@ def __init__(self, bot: Bot):
         self.bot = bot
         self.repos = []
 
+    async def cog_load(self) -> None:
+        """Function to be run at cog load."""
+        self.refresh_repos.start()
+
+        self.stored_repos_json = Path(__file__).parent.parent.parent / "resources" / "utilities" / "stored_repos.json"
```

Comment:
This variable name doesn't feel accurate. Why are they stored repos? What's special about these?

Reply:
json seems inaccurate, its a dictionary in this state.